src/database/query_builder/chainable.ts

```diff
@@ -243,6 +244,10 @@ export abstract class Chainable extends Macroable implements ChainableContract {
       return this.transformCallback(value)
     }
 
+    if (DateTime.isDateTime(value)) {
+      return value.toJSDate()
+    }
```

Comment:
Query builders never returns Luxon date times so having this conversion here is not needed